cmd/clusterctl/pkg/internal/util/yaml.go

```diff
@@ -78,6 +78,12 @@ func ToUnstructured(rawyaml []byte) ([]unstructured.Unstructured, error) {
 		var u unstructured.Unstructured
 		u.SetUnstructuredContent(m)
 
+		// Ignore empty objects.
```

Comment:
This is the same change already merged in https://github.com/kubernetes-sigs/cluster-api/pull/2326; This will go away with the next rebase